package.json

```diff
@@ -104,6 +105,7 @@
         "@types/googlemaps": "3.40.3",
         "@types/jasmine": "~3.6.2",
         "@types/jasminewd2": "~2.0.8",
+        "@types/jest": "^26.0.20",
```

Comment:
this was added otherwise VS Code complains, so prob this was not added doing the migration to NX

![image](https://user-images.githubusercontent.com/37185433/111010867-5b2ff980-835d-11eb-9597-e2745f70abb9.png)

Reply:
This is weird because we don't use jest in angular components, we use jasmine.

I think we might be running into some issues because we use jest in the stencil and jasmine in angular.